lib/custom-logger.js

```diff
@@ -26,15 +30,88 @@ function defaultRedactionRules() {
   }
 }
 
+// Function copied from: https://github.com/fastify/fastify/blob/8b8485ac5ff56d1ead387f6dd76384f996a2cf8e/lib/logger.js#L71
+function now() {
```

Comment:
@allevo this function generates a time as follows:

`{"level":30,"time":53823058.406193,"pid":41690,"hostname":"mycomputer","msg":"Server listening at http://0.0.0.0:61312","v":1}`

As you can see that's not the current time

Reply:
Taking a deeper look in pinojs I've found out how they calculate time which is just as the original [`Date.now()`](https://github.com/pinojs/pino/blob/master/lib/time.js#L5)